### PR TITLE
[FSSDK-9950] chore: SPM support added to process privacy manifest

### PR DESCRIPTION
**Pull request by muzahidul-opti**
## Summary
- SPM support added to process the privacy manifest info file. [Resource](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#resource)
- Swift tools version updated to 5.4 [About the Swift Tools Version](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#)
- Update package compiling swift version. [SwiftVersion](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#swiftversion)

## Issues
- [FSSDK-9950](https://jira.sso.episerver.net/browse/FSSDK-9950)

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -14,7 +18,11 @@ let package = Package(
                  targets: ["Optimizely"])
     ],
     targets: [
-        .target(name: "Optimizely", path: "Sources")
+        .target(
+            name: "Optimizely",
+            path: "Sources",
+            resources: [.copy("Supporting Files/PrivacyInfo.xcprivacy")]
```
Comment (charlyliu) [inline Package.swift:24]: I think you need to exclude the file and add it in the resource. I couldn't find documentation saying it but I checked many libraries that are doing that.

Comment (muzahidul-opti) [reply Package.swift:24]: > I think you need to exclude the file and add it in the resource. I couldn't find documentation saying it but I checked many libraries that are doing that.

I also haven't found any proper documentation so far. But I think exclude and added as resource same file is a kind of self-contradictory to me. I also checked it works well and didn't throw any warning or error.

Comment (charlyliu) [reply Package.swift:24]: I think the reasoning is to not add it in the source code but only inside the resource bundle. But if both works then choose whatever makes more sense for you.

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**